### DISCUSSION TITLE
Ideas for the repeater field

### DISCUSSION THREAD

**Discussion by StephenGunn** [directus/directus#4691] [Feature Requests]
Forgive me if you can do this already but I would love to:
-  Set a min / max number of items you can add
-  See the index of the item while editing

Either or both of these would be super helpful to me. The fact that I can save the data to JSON is already incredible. 

Thank you.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
